Error: bad_comment.fl(1,1): Unexpected end of input in comment 
$ foo ...
^         
